src/Umbraco.Infrastructure/Serialization/JsonFuzzyBooleanConverter.cs

```diff
@@ -0,0 +1,62 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Umbraco.Cms.Infrastructure.Serialization;
+
+/// <summary>
+/// Converts a boolean (like) value to or from JSON.
+/// </summary>
+public sealed class JsonFuzzyBooleanConverter : JsonConverter<bool>
```

Comment:
I don't think this name makes any sense. Fuzzy matching means matching something approximately, there's nothing approximately about this.

Reply:
ConvertToXConverter feels a bit repetitive. Why not just JsonBooleanConverter?